spec/models/exhibit_spec.rb

```diff
@@ -49,4 +49,35 @@
       end
     end
   end
+
```

Comment:
These look great. If you wanted, you could define the test exhibits in a `let` block at the start of the describe block, instead of creating them from scratch in each test: https://rspec.info/features/3-12/rspec-core/helper-methods/let/
If you think you might reuse the themed exhibit objects in another test file, you could even create a new factory in spec/factories/exhibits.rb.

Reply:
just updated them, hopefully its easier to read the tests now